safety/safety.py

```diff
@@ -15,14 +15,14 @@ class Vulnerability(namedtuple("Vulnerability",
     pass
 
 
-def get_from_cache(db_name):
+def get_from_cache(db_name, site_down=False):
```

Comment:
The term `site_down` does not align with a cache thing. I mean, we are already getting from cache here. I think we could pass something like `valid_seconds`.